brainframe/cli/frozen_utils.py

```diff
@@ -18,36 +18,35 @@ def defaults_file_path() -> Path:
     :return: A path to the included defaults file
     :raises FileNotFoundError: If the defaults file is not present
     """
-    if is_frozen():
-        return _pyinstaller_tmp_path() / RELATIVE_DEFAULTS_FILE_PATH
-    else:
-        return _find_resource_location(RELATIVE_DEFAULTS_FILE_PATH)
+    return _get_resource_absolute_path(RELATIVE_DEFAULTS_FILE_PATH)
 
 
 def translations_path() -> Path:
     """
     :return: A path to the folder with the translation files
     :raises FileNotFoundError: If the defaults file is not present
     """
-    if is_frozen():
-        return _pyinstaller_tmp_path() / RELATIVE_TRANSLATIONS_PATH
-    else:
-        return _find_resource_location(RELATIVE_TRANSLATIONS_PATH)
+    return _get_resource_absolute_path(RELATIVE_TRANSLATIONS_PATH)
 
 
-def _find_resource_location(target: Path) -> Path:
-    """Finds a resource given its path relative to the root of the project when the
-    application is being run from source.
+def _get_resource_absolute_path(target: Path) -> Path:
+    """Gets the absolute path of a resource.
 
     :param target: The relative path of the resource from the root of the project
-    :return: The absolute path to the resource
+    :return: The absolute path of the resource
+    :raises FileNotFoundError: If the resource cannot be found
     """
-    # Traverse up the file system from this file to find the root
-    for parent in Path(__file__).absolute().parents:
-        if (parent / target).exists():
-            return parent / target
+    if is_frozen():
+        path = _pyinstaller_tmp_path() / target
+        if not path.is_file():
+            raise FileNotFoundError()
```

Comment:
You should give this exceptions [more info](https://stackoverflow.com/a/36077407/8134178):

```python
import errno
import os

raise FileNotFoundError(errno.ENOENT, os.strerror(errno.ENOENT), filename)
```

Reply:
Usually I'm down to provide a lot of information in exceptions, but for this use-case it would all go unused always. The caller always knows what file it's asking for and would never provide the message to the user directly because we want a translated, user-readable error. If you gotta use `errno` to use `FileNotFoundError` properly, maybe I should go back to a custom exception.